accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/RestrictionMapsHolder.java

```diff
@@ -0,0 +1,47 @@
+package biz.netcentric.cq.tools.actool.helper;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.jcr.Value;
+/**
+ * Container class holding 2 maps for storing single and multivalue restrictions. singleValuedRestrictionsMap holds restriction names as keys and values of type javax.jcr.Value
+ * multiValuedRestrictionsMap holds restriction names as keys and values of type javax.jcr.Value[]
+ * @author jochenkoschorkej
+ *
+ */
+public class RestrictionMapsHolder {
```

Comment:
We should move away from maps and step towards a "configuration model" I think - what about calling it "Restrictions" or "RestrictionsModel". It can then have a List of "Restriction" classes, a Restriction could have an isArray() method.